e2e/README.md

```diff
@@ -0,0 +1,39 @@
+# Guide to Running Tests
+
+It is important to note that one must ensure that all dfusion related builds are recent (i.e. driver, graph-listener and their dependents). 
+
+From within the project source run:
+
+```shell script
+docker-compose down && docker-compose up driver graph-listener truffle
+```
+
+Once the driver has recognized the deployed contracts each of the test must be run individually as
+
+```shell script
+cargo test -p e2e snapp_<testname> -- --nocapture
+```
+The nocapture used here will display log statements while the test is running to help identify where the test is at.
+
+where `<testname>` can be any of "deposit_withdraw", "auction" or "standing_order".
+
+In order to run multiple tests some containers must be restarted and the database must be removed.
+For example, the following sequence of commands (in separate/alternating terminals) will run all the tests. 
+Notice that, although the travis file uses the bash script for restarting containers, this will not work locally.  
```

Comment:
I'm surprised by this, `restart-containers.sh` works for me it just runs them in detached mode.

Reply:
I am also surprised... since the last line of restart-containers uses that one compose file that I cannot use! Right, yes, this is a MacOS vs Linux thing I believe and this is why it works on yours.